### DISCUSSION TITLE
Is there a way to make the CLI show the whole response instead of one word at a time?

### DISCUSSION THREAD

**Discussion by BettyJJ** [waylaidwanderer/node-chatgpt-api#157] [Help] [answered]
Thank you for the project. The CLI works great, but it has this effect just like https://chat.openai.com/chat that the response appears slowly, one word at a time. Can we make it show the response all at once?

**Comment by waylaidwanderer** [comment, marked as answer]
It's shown to you as the AI model generates the token, so no. You could disable streaming but it would take the same amount of time.

**Reply by BettyJJ** [reply]
OK, I thought that was just an effect. 🤣 Thank you.

**Reply by waylaidwanderer** [reply]
Haha no problem, you're not the first person to assume this.
